alerts/thinpool-utilization.libsonnet

```diff
@@ -0,0 +1,63 @@
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'thinpool-utilization',
+        rules: [
+          {
+            alert: 'GlusterThinpoolDataUtilization',
+            expr: |||
+                  100 * gluster_thinpool_data_used_bytes{%(glusterExporterSelector)s} / gluster_thinpool_data_total_bytes{%(glusterExporterSelector)s} > 80
```

Comment:
Wouldn't `gluster_thinpool_data_used_bytes{%(glusterExporterSelector)s} / gluster_thinpool_data_total_bytes{%(glusterExporterSelector)s} > 0.8 ` work ?  Saves one operation ( 100*  multiplication here) on the metric. Same thing for other alerts.